### ISSUE TITLE
doctests are breaking nix

### ISSUE THREAD

**Issue by parsonsmatt** [pbrisbin/bugsnag-haskell#66]
Trying to build the current version of `bugsnag-haskell` from Hackage and getting a nix-related failure.

```
Linking dist/build/spec/spec ...
running tests
Running 2 test suites...
Test suite doctest: RUNNING...
doctest: InvalidYaml (Just (YamlException "Yaml file not found: package.yaml"))
Test suite doctest: FAIL
```

I think we'd want to include the `package.yaml` file in `extra-source-files` or similar so it's included in the Hackage tarball.

**Comment by pbrisbin** [comment]
Ah, thanks for the report. The intent here is to grab `default-extensions` so they don't need to be duplicated in the doctest execution.

Other fixes here could be to do that still, but from the `cabal` file, or to just not do that. It's a relatively stable list. :shrug:

**Comment by pbrisbin** [comment]
Should be fixed in v0.0.4.3: https://hackage.haskell.org/package/bugsnag-haskell-0.0.4.3/src/package.yaml
